org.eclipse.winery.repository.rest/pom.xml

```diff
@@ -209,6 +209,23 @@
             <artifactId>org.eclipse.winery.common</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <!-- begin of EDMM dependencies -->
+        <dependency>
+            <groupId>org.eclipse.winery</groupId>
+            <artifactId>org.eclipse.winery.edmm</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jgrapht</groupId>
+            <artifactId>jgrapht-core</artifactId>
+            <version>1.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.8.6</version>
```

Comment:
make sure you use the same version as Winery currently uses.

Reply:
In the root pom.xml the version specified for `com.fasterxml.jackson.core.jackson-annotations` is `2.8.0`. In this pom.xml the version used for `jackson-databind` and `jackson-core` is `2.8.6` .

Reply:
Versions need to be aligned.

Reply:
To which one? The version numbers I listed above are there not because of me. I can't touch them if I understood well. I putted here the version 2.8.6 because it's the same used in `jackson-databind` and `jackson-core`.

Reply:
Try to use version that is already "in use" by Winery.

Reply:
So, `2.8.6` everywhere, also in the `<com.fasterxml.jackson.core.jackson-annotations>` property.

Reply:
Even better, change the Maven property to `<com.fasterxml.jackson>`, set it to `2.8.6` and adapt any pom.xml file to reflect this property with `${com.fasterxml.jackson}` in the respective `<version>` tags.

Reply:
that was what I did before reverting to that. Ok, no problem I'll reset the pom.xml property.

Reply:
Now there is a property `<com.fasterxml.jackson>` in the root pom.xml. I have changed all the dependencies to jackson so that now they  use this property.